### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by banyan**

### CODE REVIEW

File: lib/sbpayment/configuration.rb [hunk 1]
```diff
@@ -17,19 +17,32 @@ class ConfigrationError < Sbpayment::Error; end
...
+      cipher_code
+      cipher_iv
+      hashkey
+      proxy_uri
+      proxy_user
+      proxy_password
+      allow_multiple_service_id
+    ].freeze
+
+    attr_accessor *OPTION_KEYS
```
Comment (kachick) [inline lib/sbpayment/configuration.rb]: Looks warnable... Is `attr_accessor(*OPTION_KEYS)` weird?

File: lib/sbpayment/configuration.rb [hunk 2]
```diff
@@ -17,19 +17,32 @@ class ConfigrationError < Sbpayment::Error; end
...
 
     def initialize
       @sandbox = false
       @allow_multiple_service_id = false
     end
 
+    def []=(name, value)
```
Comment (kachick) [inline lib/sbpayment/configuration.rb]: Looks useful :+1:

**Comment by kachick** [comment]
:+1:

**Comment by banyan** [comment]
Thanks updated: https://github.com/quipper/sbpayment.rb/commit/1f6bf52641432978486ff21c04480ae2314fe060

**Comment by kachick** [comment]
Released `v0.5.2` :sunglasses:

**Pull request merged**